[ADD] account_background_post: multiple improves

zaoral opened the pull request.

zaoral committed:
[ADD] account_background_post: auto trigger cron

If the user is using post entries action of the menu and
they select the option background post then we trigger
the cron immediately so the process of the background
invoices start.

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/account-invoicing/pull/159).

zaoral committed:
[ADD] account_background_post: action_post

Use action_post instead of post because action_post do things we
need that the _post dont: like some validations, send emails and
another payments extra steps.

In this case we hive the force_post boolean because is confusing,
now it does not do anything, all the invoices will be validated
without taking into account the accounting date

zaoral commented:
@roboadhoc rebase-ff
@roboadhoc r+

roboadhoc commented:
Merge method set to rebase and fast-forward.
